### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nh916**
I think this is slightly nicer to have the link within the readme and having the logo as a link to the website

### CODE REVIEW

**Comment by nh916** [comment]
@bearmit let me know what you think about this edit to the README.md file. I think it could be helpful. I think the UI could be slightly better, but I wasn't sure how to. Not a big deal or urgent at all, just a quick thought while looking at the repository

File: README.md
Comment (bearmit) [inline README.md:3]:
```suggestion
This is the repository of the [CRIPTScripts](https://criptscripts.org/) website
```

**Review by bearmit — APPROVED** [review]
Yes good idea. Just found a typo.

BTW, I think we will be able to redirect to [this page](https://web.mycriptapp.org/resources/scripts/) soon. Actually we could already because I converted the criptscripts home page, but articles are still pointing to https://criptscripts.org. In order to have a nice url, we could have a redirection from https://scripts.mycriptapp.org to https://web.mycriptapp.org/resources/scripts/

@Ardi028 Do you think we can start to link to the script page (of the CMS) from the readme of this repo? Or do you prefer we wait until all articles have been fully converted to the cms?

**Comment by nh916** [comment]
> Yes good idea. Just found a typo.
> 
> BTW, I think we will be able to redirect to [this page](https://web.mycriptapp.org/resources/scripts/) soon. Actually we could already because I converted the criptscripts home page, but articles are still pointing to https://criptscripts.org. In order to have a nice url, we could have a redirection from https://scripts.mycriptapp.org to https://web.mycriptapp.org/resources/scripts/
> 
> @Ardi028 Do you think we can start to link to the script page (of the CMS) from the readme of this repo? Or do you prefer we wait until all articles have been fully converted to the cms?

Wow that is awesome!

@bearmit do you think we'll still need this repo to keep all the scripts or should it be archived?

I don't know too much about this, so I will leave it to you discretion of whatever you think is best

**Comment by bearmit** [comment]
> @bearmit do you think we'll still need this repo to keep all the scripts or should it be archived?
> 
> I don't know too much about this, so I will leave it to you discretion of whatever you think is best

My opinion is we should keep the repository, but without the extra sources related to the Jekyll framework.

Maybe, having two repos would be good too (one for Python,  the other for TypeScript). The Python and TypeScript parts clearly do not have anything in common once we'll remove the Jekyll part of it.

**Comment by bearmit** [comment]
just saw I forgot to merge. Doing it now, sorry.

**Comment by nh916** [comment]
be sure to delete branch after it is merged

**Pull request merged**
